core/client/fs/src/main/java/alluxio/client/file/cache/PageMetaStore.java

```diff
@@ -26,13 +31,19 @@ public interface PageMetaStore {
    * @param conf the alluxio configuration
    * @return an instance of MetaStore
    */
-  static PageMetaStore create(AlluxioConfiguration conf) {
+  static PageMetaStore create(AlluxioConfiguration conf) throws IOException {
+    List<PageStoreDir> dirs = PageStoreDir.createPageStoreDirs(conf);
     if (conf.getBoolean(PropertyKey.USER_CLIENT_CACHE_QUOTA_ENABLED)) {
-      return new QuotaPageMetaStore(conf);
+      return new QuotaPageMetaStore(conf, dirs);
     }
-    return new DefaultPageMetaStore();
+    return new DefaultPageMetaStore(dirs);
   }
 
+  /**
+   * @return the associated lock
+   */
+  ReadWriteLock getLock();
```

Comment:
`PageMetaStore` is now shared between the cache manager and `PagedBlockStore`, so we need to share locks in the two classes.